src/AccessibilityInsights/MainWindowHelpers/StateMachine.cs

```diff
@@ -574,15 +574,16 @@ internal void HandleConfigurationChanged(IReadOnlyDictionary<string, object> cha
 
             configManager.TestConfig = TestSetting.GenerateSuiteConfiguration(RuleSelection.SuiteConfigurationType.Default);
 
-            InitSelectActionMode();
-
-            HideConfigurationMode();
-
             if (changes != null && changes.ContainsKey(ConfigurationModel.keyFontSize))
             {
                 SetFontSize();
             }
+        }
 
+        internal void TransitionToSelectActionMode()
```

Comment:
is this method used anywhere now?

Reply:
Yes it is used in buttonOk_Click ConfigurationModeControl.xaml.cs